packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.styles.ts

```diff
@@ -3,10 +3,11 @@ import { HighContrastSelector } from '@fluentui/react/lib/Styling';
 import { ISelectedPersonaStyleProps, ISelectedPersonaStyles } from './SelectedPersona.types';
 
 export const getStyles = (props: ISelectedPersonaStyleProps): ISelectedPersonaStyles => {
-  const { theme: maybeTheme, isSelected, isValid } = props;
+  const { theme: maybeTheme, isSelected, isValid, personaHeight } = props;
   // theme is actually non-nullable (supplied by styles)
   const theme = maybeTheme!;
   const { palette } = theme;
+  const buttonSize = personaHeight;
```

Comment:
Yes, I know this is unnecessary. I don't expect to check this in, as I expect to make changes as we decide (and I learn) how this should be implemented.